cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -89,7 +89,7 @@ def catalog_item(ansible_repository):
         fauxfactory.gen_alphanumeric(),
         fauxfactory.gen_alphanumeric(),
         provisioning={
-            "repository": ansible_repository.as_fill_value,
+            "repository": ansible_repository.as_fill_value(),
```

Comment:
you don't need to call as_fill_value() if you pass, this to fill further. you can just pass object itself and widgetastic will call as_fill_value.

Reply:
Doesn't seem that the fill call is picking this up as an object and calling as_fill_value on it, it's getting passed as an object resulting in an error when a replace method is called (expecting string in fill)

Reply:
Repository is not inherited from Fillable. it should start to work if you add it

Reply:
@izapolsk pointed out offline that I hadn't added Fillable inheritence.